app/models/lunch.rb

```diff
@@ -56,31 +67,39 @@ def chalmrest
-        {name: "Kårrestaurangen", url: "http://intern.chalmerskonferens.se/view/restaurant/karrestaurangen/Veckomeny.rss?today=true&locale=#{locale}", location: "Johanneberg"},
-        {name: "L's kitchen", url: "http://intern.chalmerskonferens.se/view/restaurant/l-s-kitchen/Projektor.rss?today=true&locale=#{locale}", location: "Lindholmen"},
-        {name: "Express", url: "http://intern.chalmerskonferens.se/view/restaurant/express/V%C3%A4nster.rss?today=true&locale=#{locale}", location: "Johanneberg"},
+        {name: "Linsen", url: "http://carboncloudrestaurantapi.azurewebsites.net/api/menuscreen/getdataday?restaurantid=33", location: "Johanneberg"},
+        {name: "Kårrestaurangen", url: "http://carboncloudrestaurantapi.azurewebsites.net/api/menuscreen/getdataday?restaurantid=5", location: "Johanneberg"},
+        {name: "L's kitchen", url: "http://carboncloudrestaurantapi.azurewebsites.net/api/menuscreen/getdataday?restaurantid=8", location: "Lindholmen"},
+        {name: "Express", url: "http://carboncloudrestaurantapi.azurewebsites.net/api/menuscreen/getdataday?restaurantid=9", location: "Johanneberg"},
         #{name: "J.A Pripps", url: "http://intern.chalmerskonferens.se/view/restaurant/j-a-pripps-pub-cafe/RSS%20Feed.rss?today=true&locale=#{locale}", location: "Johanneberg"},
         #{name: "Restaurang Hyllan", url: "http://intern.chalmerskonferens.se/view/restaurant/hyllan/RSS%20Feed.rss?today=true&locale=#{locale}", location: "Johanneberg"},
-        {name: "L's Resto", url: "http://intern.chalmerskonferens.se/view/restaurant/l-s-resto/RSS%20Feed.rss?today=true&locale=#{locale}", location: "Lindholmen"},
-        {name: "Kokboken", url: "http://intern.chalmerskonferens.se/view/restaurant/kokboken/RSS%20Feed.rss?today=true&locale=#{locale}", location: "Lindholmen"}
+        {name: "L's Resto", url: "http://carboncloudrestaurantapi.azurewebsites.net/api/menuscreen/getdataday?restaurantid=32", location: "Lindholmen"},
+        {name: "Kokboken", url: "http://carboncloudrestaurantapi.azurewebsites.net/api/menuscreen/getdataday?restaurantid=35", location: "Lindholmen"}
       ]
 
       restaurants.map do |restaurant|
-        meals = Feed.fetch_and_parse(restaurant[:url]).entries.map do |entry|
-          summary, price = entry.summary.split('@')
-          summary = summary.strip
-          price = price.strip
-          if restaurant[:name] == "Express" || restaurant[:name] == "Kårrestaurangen"
-            allergens = get_allergens(entry)
-          end
-          unless summary.empty?
-            { title: entry.title, summary: summary, price: price.try(&:to_i), allergens: allergens }
-          end
-        end.compact
+        puts restaurant
+        data = JSON.parse(open(restaurant[:url]).read)
+
+        if data.key?("recipeCategories") && data["recipeCategories"] != nil
```

Comment:
The same goes for other string literals. "recipes", "displayNames", "allergens" etc...